src/api/spec/models/obs_factory/distribution_strategy_sle12_sp1_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe ObsFactory::DistributionStrategySLE12SP1 do
+  let(:project) { create(:project, name: 'SUSE:SLE-12-SP1:GA') }
+  let(:project_staging_a) { create(:project, name: 'SUSE:SLE-12-SP1:GA:Staging:A') }
+  let(:distribution) { ObsFactory::Distribution.new(project) }
+  let(:staging_project) { ObsFactory::StagingProject.new(project: project_staging_a, distribution: distribution) }
+
+  subject { distribution.strategy }
+
+  describe '#sp_version' do
+    it { expect(subject.sp_version).to eq('SP1') }
+  end
+
+  describe '#openqa_version' do
+    it { expect(subject.openqa_version).to eq('SLES 12 SP1') }
+  end
+
+  describe '#openqa_iso' do
+    let(:iso_filename) { 'ISO-SUSE-SLE-12.x86_64-SP1-Build16.64.iso' }
+    before do
+      allow_any_instance_of(ObsFactory::DistributionStrategySLE12SP1).to receive(:project_iso).and_return(iso_filename)
```

Comment:
now that looks wrong. Please use some real world example from SP4's A. If you check, the isos are rather named Test-Server-DVD-x86_64-Build19.2-Media.iso and are queried on openqa as SLE12-SP4-Staging:A-Test-Server-DVD-x86_64-BuildA.19.2-Media.iso

Reply:
I don't mind, but it's really SP1+ - what I said was true for SP1 as well.